Makefile

```diff
@@ -266,6 +268,11 @@ minikube-deploy: minikube-start gadget-default-container kubectl-gadget
 btfgen:
 	+make -f Makefile.btfgen
 
+.PHONY: check_build_requirements
+check_build_requirements:
+	@which go > /dev/null && go version | grep $(GOLANG_VERSION) || echo 'You need to have golang version $(GOLANG_VERSION) to build kubectl_gadget' >&2
```

Comment:
AFAIU that's the minimal golang version. For instance, the current code should compile fine with golang 1.20.